app/src/main/java/fr/free/nrw/commons/utils/BetaCheckerUtil.java

```diff
@@ -0,0 +1,10 @@
+package fr.free.nrw.commons.utils;
+
+import fr.free.nrw.commons.BuildConfig;
+
+public class BetaCheckerUtil {
```

Comment:
I feel `BetaCheckerUtil` is quite restrictive in its naming. Would `BuildConfigUtils` be better naming? Open to any other names as well that are less restrictive about what the class can provide, or expand to in future.

Reply:
I agree with you there, as and when the class starts to house functions with make use of other constants from BuildConfig, the class name should be appropriately refactored. But since it was only using this bit, I let SRP set the name.

Reply:
Yup, I agree with @sup95. Let us rename the class.